### ISSUE TITLE
Add line numbers

### ISSUE THREAD

**Issue by jmccay-work** [rainman74/NPPTextFX2#25]
Is it possible to modify how the line numbers are formatted when using add line numbers?

**Comment by rainman74** [comment]
I don't think you need more than a numerical list, so no, nothing further is planned here.
